src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3675,6 +3675,56 @@ public sealed partial class WeakReference<T> : System.Runtime.Serialization.ISer
         public void SetTarget(T target) { }
         public bool TryGetTarget(out T target) { throw null; }
     }
+
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Auto)]
+    public partial struct HashCode
+    {
+        private uint _v1, _v2, _v3, _v4;
+        private uint _queue1, _queue2, _queue3;
+        private uint _length;
```

Comment:
Private fields aren't included in the refs for other structs (e.g. `DateTime`), why are they needed here?

Reply:
As per the rest of the file, it seems as though either the fields are present or `[StructLayout(Size))]` is set. Maybe the length of these types is important?